src/dexpo.cpp

```diff
@@ -5,32 +5,16 @@
 #include "wmctrl.hpp"
 #include <unistd.h>
 #include <vector>
-desktop_pixmap d1 (0, 0, 1920, 1080, "DP-1");
-desktop_pixmap d2 (0, 0, 1920, 1080, "DP-2");
-desktop_pixmap d3 (0, 0, 1920, 1080, "DP-3");
 
-// Used for testing purposes
-std::vector<dexpo_pixmap>
-test_get_vector ()
-{
-  d1.save_screen ();
-  d2.save_screen ();
-  d3.save_screen ();
-
-  dexpo_pixmap d1_1{ 0, d1.pixmap_width, d1.pixmap_height, d1.pixmap_id,
-                     "DP-1" };
-  dexpo_pixmap d2_1{ 1, d2.pixmap_width, d2.pixmap_height, d2.pixmap_id,
-                     "DP-2" };
-  dexpo_pixmap d3_1{ 2, d3.pixmap_width, d3.pixmap_height, d3.pixmap_id,
-                     "DP-3" };
-  return std::vector<dexpo_pixmap>{ d1_1, d2_1, d3_1 };
-}
+// TODO Holy fuck we need to fix this burning shit
```

Comment:
No swearing in comment

Reply:
I am the DevOps here. My code my rules. xD